server/services/user.js

```diff
@@ -13,6 +14,50 @@ const getUser = async (id) => {
   return dbClient.db[collections.USERS].findDoc(query, options);
 };
 
+/**
+ * Returns all ROS statuses for given sites that are currently 'hired' (not archived) that started a year ago or more
+ * @param {[id: number]} sites
+ * @returns [ROS_STATUS join to PARTICIPANTS_STATUS]
+ */
+const getROSEndedNotifications = async (sites) => {
+  const todayDate = dayjs(new Date()).subtract(1, 'y').format('YYYY-MM-DD');
+  return dbClient.db[collections.ROS_STATUS]
+    .join({
+      participantStatus: {
+        relation: collections.PARTICIPANTS_STATUS,
+        type: 'LEFT OUTER',
+        on: {
+          participant_id: 'participant_id',
+        },
+      },
+    })
+    .find({
```

Comment:
```suggestion
    .find({
    'data.date::timestamp <': todayDate,
    'participantStatus.current': true,
    'participantStatus.status': 'hired',
    'participantStatus.data.site IN': sites
    })
```
1. No need for `and`
2. `participantStatus.data.site` should `IN` `sites`

Reply:
Getting rid of the and!
The "IN" isn't necessary though, Massive defaults to "IN" as an operator just like equality when it's an array (https://massivejs.org/docs/criteria-objects#scalar-comparison)

Reply:
ok